features/formatter_paths.feature

```diff
@@ -3,15 +3,15 @@ Feature: Formatter Paths
   Background:
     Given a file named "features/a.feature" with:
       """
-      Feature: some feature
-        Scenario: some scenario
-          Given a passing step
+Feature: some feature
```

Comment:
Ooops @eoola we shouldn't have pushed these reformatting changes.

Reply:
It was probably your editor doing an autoformat on the file I guess.

Can you see if you can push another commit to the branch that puts the indentation of these lines back to what it was before?